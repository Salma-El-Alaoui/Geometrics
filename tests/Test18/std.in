C c1 1 1 1
R r2 2 2 2 2
L l3 3 3 3 3
PL pl4 4 4 4 4 4 4
OA oa5 c1 r2 l3 pl4
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
REDO
LIST
REDO
LIST
REDO
LIST
REDO
LIST
REDO
EXIT

