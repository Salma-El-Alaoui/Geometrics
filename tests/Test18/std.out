OK
#Element ajoute: c1
OK
#Element ajoute: r2
OK
#Element ajoute: l3
OK
#Element ajoute: pl4
OK
#Element ajoute: oa5
C c1 1 1 1
L l3 3 3 3 3
OA oa5 c1 l3 pl4 r2
PL pl4 4 4 4 4 4 4
R r2 2 2 2 2
OK
#Undone
C c1 1 1 1
L l3 3 3 3 3
PL pl4 4 4 4 4 4 4
R r2 2 2 2 2
OK
#Undone
C c1 1 1 1
L l3 3 3 3 3
R r2 2 2 2 2
OK
#Undone
C c1 1 1 1
R r2 2 2 2 2
OK
#Undone
C c1 1 1 1
OK
#Undone
OK
#Redone
C c1 1 1 1
OK
#Redone
C c1 1 1 1
R r2 2 2 2 2
OK
#Redone
C c1 1 1 1
L l3 3 3 3 3
R r2 2 2 2 2
OK
#Redone
C c1 1 1 1
L l3 3 3 3 3
PL pl4 4 4 4 4 4 4
R r2 2 2 2 2
OK
#Redone

